test-0041.c:2:5: warning: support of call recursion is not stable yet
test-0041.c:2:5: note: nestLevel is 2
test-0041.c:1:5: error: call cache entry found, but result not computed yet; perhaps a recursive function call?
test-0041.c:1:5: warning: end of function main() has not been reached
test-0041.c:2:5: note: from call of main()
test-0041.c:1:5: note: from call of main()
